plugwise/smile.py

```diff
@@ -320,38 +317,31 @@ async def _smile_detect_legacy(self, result: etree, dsmrmain: etree) -> str:
         if result.find('./appliance[type="thermostat"]') is None:
             # It's a P1 legacy:
             if dsmrmain is not None:
-                try:
-                    status = await self._request(STATUS)
-                    self.smile_fw_version = status.find("./system/version").text
-                    model = status.find("./system/product").text
-                    self.smile_hostname = status.find("./network/hostname").text
-                    self.smile_mac_address = status.find("./network/mac_address").text
-                except InvalidXMLError:  # pragma: no cover
```

Comment:
agreed on removing this, but I recall we really had a p1 corner case, lets see if it reoccurs